app/views/layouts/_navbar.html.erb

```diff
@@ -70,6 +71,8 @@
 
       <li role="presentation"><%= link_to "Bulk Email Instructors", email_instructors_admin_path, title: "Send a bulk email to every Autolab instructor" %></li>
 
+      <li role="presentation"><%= link_to "Clear Cache", clear_cache_admin_url, title: "Clear Cache" %></li>
```

Comment:
Add `, data: {method: "post"}` at the end to make it work with `post` requests (see below comment)

i.e. `link_to "Clear Cache", clear_cache_admin_url, title: "Clear Cache", data: {method: "post"}`

Reply:
Fixed.